src/api/annotation/AnnotationAPI.js

```diff
@@ -146,14 +152,28 @@ export default class AnnotationAPI extends EventEmitter {
         this.openmct.objects.mutate(domainObject, this.ANNOTATION_LAST_CREATED, Date.now());
     }
 
+    /**
+    * @method defineTag
+    * @param {String} key a unique identifier for the tag
+    * @param {Tag} tagsDefinition the definition of the tag to add
+    */
     defineTag(tagKey, tagsDefinition) {
         this.availableTags[tagKey] = tagsDefinition;
     }
 
+    /**
+    * @method isAnnotation
+    * @param {module:openmct.DomainObject} domainObject the domainObject in question
```

Comment:
Does using `module:` work? I found I had to [@import](https://github.com/nasa/openmct/blob/master/src/api/user/StatusUserProvider.js#L53) type definitions like this.

Reply:
Heh, this is a dumb question, but are we using JSDoc or Typescript for code annotations? I ask, because most of the APIs look like they're generally using `module` (JSDoc), e.g., in [Object](https://github.com/nasa/openmct/blob/master/src/api/objects/ObjectAPI.js#L62) and [Composition](https://github.com/nasa/openmct/blob/master/src/api/composition/CompositionAPI.js#L59). Elsewhere, we're using Typscript (e.g., the [Status User](https://github.com/nasa/openmct/blob/master/src/api/user/StatusUserProvider.js#L53) stuff). Also, is there a canonical way of testing the compilation of the code annotations?

Reply:
We're switching to Typescript per @akhenry